vm/src/stdlib/json.rs

```diff
@@ -64,7 +64,7 @@ impl<'s> serde::Serialize for PyObjectSerializer<'s> {
             serialize_seq_elements(serializer, &elements)
         } else if objtype::isinstance(self.pyobject, &self.vm.ctx.dict_type()) {
             let dict: PyDictRef = self.pyobject.clone().downcast().unwrap();
-            let pairs = dict.get_key_value_pairs();
+            let pairs: Vec<(PyObjectRef, PyObjectRef)> = dict.into_iter().collect();
```

Comment:
nit: the item type is implied by the dict, so this could just be `let pairs: Vec<_> = ...`